### PR TITLE
Product change

### PR DESCRIPTION
**Pull request by tomjur2107**

### CODE REVIEW

File: PhotoPlusBackend/photoplus/.idea/misc.xml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="JavaScriptSettings">
+    <option name="languageLevel" value="ES6" />
```
Comment (syphzur) [inline PhotoPlusBackend/photoplus/.idea/misc.xml]: ????

File: PhotoPlusFrontend/PhotoPlus/src/app/components/product/product.component.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<div class="wrapper fadeInDown" >
+    <p class="card-subtitle mb-2 text-muted"> {{ product.category }} </p>
+
+    <p class="name" >{{product.name}}</p>
+    
+    
+    
+    <ngb-carousel wrap="true" style="width:30%;margin-left: 5%;float: left;" [interval]="false" *ngIf="product.imagesUrl.length > 1; else noCarousel" #carousel>
```
Comment (syphzur) [inline PhotoPlusFrontend/PhotoPlus/src/app/components/product/product.component.html:8]: i think you should move style to .scss file

File: PhotoPlusFrontend/PhotoPlus/src/app/components/product/product.component.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        <ng-template ngbSlide *ngFor="let image of product.imagesUrl">
+          <div class="picsum-img-wrapper" >
+            <img class="carouselImage" src="{{image}}">
+          </div>
+        </ng-template>
+      </ngb-carousel>
+      <ng-template #noCarousel  >
+        <div style="width:30%;margin-left: 5%;float: left;">
```
Comment (syphzur) [inline PhotoPlusFrontend/PhotoPlus/src/app/components/product/product.component.html:16]: i think you should move style to .scss file

File: PhotoPlusFrontend/PhotoPlus/src/app/components/product/product.component.html (hunk 2 above)
Comment (syphzur) [reply PhotoPlusFrontend/PhotoPlus/src/app/components/product/product.component.html:8]: @Suchar230 please check it

File: PhotoPlusFrontend/PhotoPlus/src/app/components/product/product.component.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+          <img class="noCarouselImage" src="{{product.imagesUrl[0]}}">
+        </div>    
+      </ng-template>
+     <p class="description"> {{ product.description }} </p>
+    
+    
+      <p class="price" style="float:left" >$<span>{{ product.price }}</span></p>
```
Comment (Suchar230) [inline PhotoPlusFrontend/PhotoPlus/src/app/components/product/product.component.html:23]: !

**Review by dsuchy — CHANGES_REQUESTED** [review]
Styling being done in both scss and html files.
Html styling being done even for elements already styled in scss, overwriting scss values

File: PhotoPlusFrontend/PhotoPlus/src/app/components/product/product.component.html (hunk 2 above)
Comment (dsuchy) [reply PhotoPlusFrontend/PhotoPlus/src/app/components/product/product.component.html:8]: Html styling when scss already in use risks unexpected behaviour and makes code unmaintainable

**Pull request closed without merging**